apps/web/src/lib/middleware-utils.ts

```diff
@@ -1,10 +1,8 @@
 import { cookies, headers as nextHeaders } from 'next/headers';
-import 'server-only';
```

Comment:
why remove this?

Reply:
ah I think I accidentally removed it, adding it back